diagram/SetService.js

```diff
@@ -0,0 +1,22 @@
+export default class SetService{
```

Comment:
```javascript
class SetService{}
```
then
```javascript
export default new SetService();
```
allows you to use the same object in the entire app, like this:
```javascript
import setService from 'SetService'; // lowercase setService (object)
```

Reply:
I actually do take care of the export in the next PR (or maybe the third one? I can’t remember but it does turn into a default export at some point along the way) but I’m all for moving it to lowercase and trying to keep the convention that only classes are capitalized.

Reply:
Ohhh, sorry, I was reading this on my phone earlier and it was hard to tell what the suggestion was. Yes, this change makes sense - there's no reason to have to create new instances of the object in each file it's imported into.